argoverse/evaluation/detection/utils.py

```diff
@@ -212,10 +214,6 @@ def assign(dts: np.ndarray, gts: np.ndarray, cfg: DetectionCfg) -> np.ndarray:
             of true positive thresholds used for AP computation and S is the number of true positive errors.
     """
 
-    # Ensure the number of boxes considered per class is at most `MAX_NUM_BOXES`.
```

Comment:
Are we removing this intentionally and does the downstream code handle this change well?

Reply:
Nvm, you have handled it below.